unfoldingWord/translationCore
# Implement flex layouts

#### This pull request addresses:

Issue #1168 is basically just to make the save and continue buttons show up on the screen when the window is smaller, but these PR's do a full overhaul of the styling and layout of the tools components using flex.



#### How to test this pull request:

Open a project in notes or words and look at the screen at max size and min size and anywhere in between.  The elements should always be visible on the screen and should flex to fill the screen as available.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1643)
